viewer/packages/camera-manager/tests/DefaultCameraManager.test.ts

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { DefaultCameraManager } from '../src/DefaultCameraManager';
+import { InputHandler } from '@reveal/utilities';
+
+describe('DefaultCameraManager', () => {
```

Comment:
```suggestion
describe(DefaultCameraManager.name, () => {
```

This test could be extensively extended but I'm not sure it's worth it at the current stage.

Reply:
Totally agree, I could add some tests in another PR.